pkg/kapp/diff/resource_with_history.go

```diff
@@ -80,6 +80,19 @@ func (r ResourceWithHistory) RecordLastAppliedResource(appliedChange Change) (ct
 
 	diff := appliedChange.OpsDiff()
 
+	const resourceSizeMaxLen = 262144
+
+	// kapp deploy should work without adding disable annotation when resource size exceed
+	// (https://github.com/vmware-tanzu/carvel-kapp/issues/410)
+	if len(appliedResBytes) > resourceSizeMaxLen || len(diff.MinimalMD5()) > resourceSizeMaxLen {
+		if resourceWithHistoryDebug {
+			fmt.Printf("Unable to record resource, exceed limit of %d bytes "+
```

Comment:
I would still insist that this information is made available to the users, in a way that gives them an idea of how this affects their experience

Reply:
Yeah, we can take inputs from others also on this.